src/components/dropdown/interfaces.ts

```diff
@@ -9,7 +9,7 @@ export interface OptionRendererProps<T> {
 
 export type HTMLInputProps = Omit<
   React.HTMLProps<HTMLInputElement>,
-  'onChange' | 'value'
+  'onChange' | 'value' | 'size' | 'defaultValue'
```

Comment:
are these conflicting also?

Reply:
yes, same issue

Reply:
I am wondering if this was also the root cause of this issue https://perzoinc.atlassian.net/browse/APP-4637 😂